sklearn/utils/__init__.py

```diff
@@ -1182,3 +1183,302 @@ def is_abstract(c):
     # itemgetter is used to ensure the sort does not extend to the 2nd item of
     # the tuple
     return sorted(set(estimators), key=itemgetter(0))
+
+
+def _empty_metadata_request():
+    return Bunch(fit={},
```

Comment:
What about `partial_fit`?

Reply:
Would it make sense to create this automatically if `self` is provided and otherwise, return a default?

Reply:
Creating from `self` would allow having an explicit initialization in the `_MetaRequest` class I think.

Reply:
If we had a metaestimator that knew how to call `partial_fit` on its child, I'd think it should comply with the "fit" request.